package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "road2",
-  "version": "2.0.0-DEVELOP",
+  "version": "2.0.0",
```

Comment:
it is the only place you find a version. Like for r2gg, can you keep the flag on develop ? :)

Reply:
ok !